scraper/go.mod

```diff
@@ -7,6 +7,7 @@ require (
 	go.opentelemetry.io/collector/component v1.42.0
 	go.opentelemetry.io/collector/component/componenttest v0.136.0
 	go.opentelemetry.io/collector/pdata v1.42.0
+	go.opentelemetry.io/collector/pdata/pprofile v0.136.1-0.20250926084501-6ccdc890b16f
```

Comment:
You should add a replace statement at the end of the file, so you don't rely on a released version.